### ISSUE TITLE
[Accessibility]: Incorrect control type of dropdown menu items

### ISSUE THREAD

**Issue by vladimir-krestov** [dotnet/winforms#1582]
* .NET Core Version: 3.0.100-preview8-013392
* Have you experienced this same bug with .NET Framework?: Yes

**Original bug:**
- [Bug 937670: ](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/937670)[Screen Reader - SSMA - Create new project] Incorrect control type is defined for all menus like 'File and Edit and View and Tools...' on SSMA.

**Problem description:**
- ToolStrip menu items have incorrect Accessibility ControlType property value when they are expanded. ToolStrip behavior should be like as in the Notepad.

**Actual behavior:** 
- ToolStripDropDown item doesn't change his ControlType property value when it is expanded
- The navigation tree has an extra "toolbar".
- The Inspect can't get access to item's AccessibleObject when the mouse is over it

![image](https://user-images.githubusercontent.com/49272759/62624995-04667080-b92d-11e9-8797-390fd59c030e.png)

**Expected behavior:**
- ToolStripDropDown item has "menu item" control type when it is collapsed.
- ToolStripDropDown item has "menu" control type when it is expanded and has separate space in the navigation tree.
- The navigation tree doesn't have an extra "toolbar".
- The Inspect can get access to item's AccessibleObject when the mouse is over it

![image](https://user-images.githubusercontent.com/49272759/62625513-05e46880-b92e-11e9-8c82-050bebe9f81e.png)

**Minimal repro:**
- Create Core test app
- Add MenuStrip control
- Add menu items to MenuStrip control
- Add nested items to some dropdown MenuStrip item
- Open the Inspect and check the accessibility navigation tree

**Comment by Jessie-Zhang01** [comment]
Verified this bug in .Net Core SDK 3.0.100-Preview 9-13959 build from release branch. This bug is fixed. Please see following screenshot:
![image](https://user-images.githubusercontent.com/45864649/63499061-ac696580-c4f9-11e9-9974-637a1c0399e2.png)

**Comment by RussKie** [comment]
Thank you @Jessie-Zhang01
